client/web/compose/src/components/ModuleFields/Editor/File.vue

```diff
@@ -49,6 +64,62 @@
       mode="list"
       class="mt-2"
     />
+
+    <b-modal
+      ref="modal"
+      size="lg"
+      modal-class="video-modal-width"
+      :title="$t('editor.file.webcam.title')"
+      ok-only
+      ok-title="Close"
+      body-class="d-flex flex-column align-items-center"
+      @ok="closeCamera"
+      @show="initializeWebcam"
+    >
+      <video
+        ref="video"
+        width="640"
+        height="480"
+        autoplay
+        style="display:none;"
+      />
+      <canvas
+        ref="canvas"
+        width="640"
+        height="480"
+        style="display:none;"
+      />
+      <template #modal-footer>
+        <div class="d-flex align-items-center gap-2">
+          <b-button
+            ref="closeButton"
+            variant="light"
+            @click="closeCamera"
+          >
+            {{ $t('editor.file.webcam.buttons.cancel') }}
+          </b-button>
+
+          <b-button
+            v-if="!hasCapturedImage"
```

Comment:
This doesnt have to be 2 buttons. 
Instead have one button and change its props based on hasCapturedImage